fe/fe-core/src/main/java/org/apache/doris/job/extensions/mtmv/MTMVTask.java

```diff
@@ -253,9 +258,18 @@ public synchronized void onSuccess() throws JobException {
     @Override
     protected synchronized void executeCancelLogic() {
         LOG.info("mtmv task cancel, taskId: {}", super.getTaskId());
+        // If the be scheduling has not been triggered yet, cancel the scheduling first
+        if (command != null) {
+            command.cancel();
```

Comment:
could we do insert overwrite cancel in executor.cancel(). and wait real cannel in insert overwrite cancel? current impl is very hard to use